PostmanCollections/Payment Hub.json

```diff
@@ -44974,6 +44974,638 @@
 					"response": []
 				}
 			]
+		},
+		{
+			"name": "Paybill APIs",
```

Comment:
we need to check this way of categorizing the APIs in the documentation

Reply:
We need the following changes.
1. mpesa connection apis(validation and confirmation) to Mpesa APIs folder
2. the account status API goes to the channel connector.
3.Paygops and Pesa core APis goes to Account Management folder